react/src/components/masthead/components/SprkMastheadAccordionItem/SprkMastheadAccordionItem.test.js

```diff
@@ -161,31 +157,34 @@ describe('SprkMastheadAccordionItem:', () => {
 
     const menuItemContent = wrapper.find('.sprk-c-MastheadAccordion__details');
     const menuItemContentId = menuItemContent.getDOMNode().getAttribute('id');
-    const menuItemToggleAriaControls = menuItemToggle.getDOMNode().getAttribute('aria-controls');
+    const menuItemToggleAriaControls = menuItemToggle
+      .getDOMNode()
+      .getAttribute('aria-controls');
 
     expect(menuItemContentId).toMatch(/sprk_accordion_item_\d/);
     expect(menuItemToggleAriaControls).toEqual(menuItemContentId);
     expect(wrapper.find(`#${menuItemContentId}`).length).toBe(1);
   });
 
-  it('should add correct aria-controls and id to narrowNav if narrowNavId is passed', () => {
-    const testItemId = "test12345";
+  it(`should add correct aria-controls and id to
+  narrowNav if narrowNavId is passed`, () => {
+    const testItemId = 'test12345';
     const wrapper = mount(
       <SprkMastheadAccordionItem
         text="Item 1"
         itemId={testItemId}
-        subNavLinks={[{ text: 'Item 1'}]}
+        subNavLinks={[{ text: 'Item 1' }]}
       />,
     );
 
     const menuItemToggle = wrapper.find('.sprk-c-MastheadAccordion__summary');
     const menuItemContent = wrapper.find('.sprk-c-MastheadAccordion__details');
     const menuItemContentId = menuItemContent.getDOMNode().getAttribute('id');
-    const menuItemToggleAriaControls = menuItemToggle.getDOMNode().getAttribute('aria-controls');
+    const menuItemToggleAriaControls = menuItemToggle
+      .getDOMNode()
+      .getAttribute('aria-controls');
 
     expect(menuItemContentId).toEqual(testItemId);
     expect(menuItemToggleAriaControls).toEqual(testItemId);
   });
```

Comment:
Great job on this fix! It looks like we already have tests to make sure a provided `id` is not overwritten by a calculated `id` - can we get a new test to make sure the id doesn't change when we toggle?